test_infra/test_infra/base_stack.py

```diff
@@ -0,0 +1,129 @@
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_glue as glue
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_kms as kms
+from aws_cdk import aws_logs as logs
+from aws_cdk import aws_s3 as s3
+from aws_cdk import core as cdk
+
+
+class BaseStack(cdk.Stack):  # type: ignore
+    def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs: str) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        self.vpc = ec2.Vpc(
+            self,
+            "aws-data-wrangler-vpc",
+            cidr="11.19.224.0/19",
+            enable_dns_hostnames=True,
+            enable_dns_support=True,
+        )
+        self.key = kms.Key(
+            self,
+            id="aws-data-wrangler-key",
+            description="Aws Data Wrangler Test Key.",
+            policy=iam.PolicyDocument(
+                statements=[
+                    iam.PolicyStatement(
+                        sid="Enable IAM User Permissions",
+                        effect=iam.Effect.ALLOW,
+                        actions=["kms:*"],
+                        principals=[iam.AccountRootPrincipal()],
+                        resources=["*"],
+                    )
+                ]
+            ),
+        )
+        kms.Alias(
+            self,
+            "aws-data-wrangler-key-alias",
+            alias_name="alias/aws-data-wrangler-key",
+            target_key=self.key,
+        )
+        self.bucket = s3.Bucket(
+            self,
+            id="aws-data-wrangler",
+            versioned=True,
```

Comment:
Is the bucket versioned in the CFN definition? The life cycle policy deletes everything after 1 day anyways, but I don't see why versioning is required?

Reply:
i remember I found this strange, too, but I don't see it now :) I was probably looking somewhere else. Will remove that one.